workflow/executor/k8sapi/k8sapi.go

```diff
@@ -41,6 +41,15 @@ func (k *K8sAPIExecutor) GetOutputStream(containerID string, combinedOutput bool
 	return k.client.getLogsAsStream(containerID)
 }
 
+func (k *K8sAPIExecutor) GetExitCode(containerID string) (int32, error) {
```

Comment:
Likewise -- why `int32` as opposed to `int`?  Also should return `0` when errored